src/Row.js

```diff
@@ -156,7 +156,7 @@ export default class Row extends Component {
     return (
       <Animated.View
         {...this._panResponder.panHandlers}
-        style={[style, styles.container, this._animatedLocation.getLayout()]}
+        style={[style, styles.container, {transform: this._animatedLocation.getTranslateTransform()}]}
```

Comment:
Actually it seems like rnrf allows my other scene's render to still be called while I'm in the ListView scene. After minimizing the render calls the performance for sortable-list is better. Maybe this fix is only useful under more strenuous conditions?